src/Pim/Bundle/DataGridBundle/Extension/MassAction/MassActionDispatcher.php

```diff
@@ -39,22 +39,28 @@ class MassActionDispatcher
     /** @var MassActionParametersParser */
     protected $parametersParser;
 
+    /** @var array */
+    protected $gridsWithoutMassActionRepository;
```

Comment:
Why suffixing it with "Repository"? Just `$gridsWithoutMassAction` seems enough to me.